solr-plugin/src/test/java/org/entando/entando/plugins/jpsolr/web/content/AdvContentSearchControllerTest.java

```diff
@@ -1143,6 +1145,9 @@ void testSearchByOption() throws Exception {
                     "en");
             this.contentManager.insertOnLineContent(content2);
             content2Id = content2.getId();
+            synchronized (this) {
```

Comment:
same as previous this could lead to a problem, what if the pipeline is slower than usual?